op5build/check_k8s.spec

```diff
@@ -1,5 +1,8 @@
 %define exec_path check_k8s.py
-%define pkg_path k8s
+%define app_install_path /opt/monitor/op5/check_k8s
```

Comment:
You might want to rethink the dir strucure here. 
If we include all python plugins as venv and use /opt/monitor/op5 as install path, this dir could get cluttered